### PR TITLE
fix(/learn): Made changes to increase contrast of offline warning banner

### LINKED ISSUE #40193: Offline warning message is hard to read in dark mode
**To Reproduce**
1. Go to /learn or /settings with the theme set to dark
2. Turn off internet connection and wait

**Actual behavior**
The offline warning message shows up, but the text is quite hard to read.

**Screenshots**

Screen Shot 2020-11-09 at 12 22 16 AM

### PR DESCRIPTION
**Pull request by nullaf**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #40193

### CODE REVIEW

File: client/src/components/OfflineWarning/OfflineWarning.js [hunk 1]
```diff
@@ -26,7 +26,9 @@ function OfflineWarning({ isOnline, isSignedIn }) {
 
   return showWarning ? (
     <div className='offline-warning'>
```
Comment (nhcarrigan) [inline client/src/components/OfflineWarning/OfflineWarning.js:28]:
```suggestion
    <div className='offline-warning alert-info'>
```

File: client/src/components/OfflineWarning/OfflineWarning.js [hunk 2]
```diff
@@ -26,7 +26,9 @@ function OfflineWarning({ isOnline, isSignedIn }) {
...
-      You appear to be offline, your progress may not be being saved.
+      <div className='alert-info'>
+        You appear to be offline, your progress may not be being saved.
+      </div>
```
Comment (nhcarrigan) [inline client/src/components/OfflineWarning/OfflineWarning.js:31]:
```suggestion
      You appear to be offline, your progress may not be being saved.
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
This should be kept in one div, not a nested div, I believe.

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! Pulled this down locally, tested the offline notice. Contrast is at 4.53 now, which puts it within AA standard.

Congratulations on your first pull request~! We look forward to future contributions!

Please note that our workflow requires at least one additional approval before this can be merged. 

Additionally, I recommend reviewing our [contributing guidelines](https://contribute.freecodecamp.org), specifically the section regarding creating a new branch for your work - it is, in general, a good idea to keep your `master` branch clean. 🙂

**Review by ahmaxed — APPROVED** [review]

**Pull request merged**